ampersand-dom-bindings.js

```diff
@@ -35,7 +40,7 @@ function switchHandler(binding, el, value) {
     for (var item in binding.cases) {
         var curValue = binding.cases[item];
         if (value !== item && curValue !== showValue) {
-            getMatches(el, curValue).forEach(function (match) {
+            getMatches(el, curValue, binding.firstMatchOnly).forEach(function (match) {
```

Comment:
Don't you need to add the firstOnly parameter to the getmatches call below this? Otherwise you would be hiding only the first element found for the nonshowing cases, but showing all elements found for the case to be shown.